strapi/src/api/tax-administrators-list/content-types/tax-administrators-list/schema.json

```diff
@@ -0,0 +1,21 @@
+{
+  "kind": "singleType",
+  "collectionName": "tax_administrators",
+  "info": {
+    "singularName": "tax-administrators-list",
+    "pluralName": "tax-administrators",
```

Comment:
I would use name `taxAdministratorsList` in GUI and leave Strapi to generate others.

Reply:
Fixed.